src/gauge/radial-gauge/radial-gauge.component.ts

```diff
@@ -0,0 +1,266 @@
+import { Component, AfterViewInit, ViewEncapsulation, 
+  ChangeDetectionStrategy, Input, Output, EventEmitter } from '@angular/core';
+
+import { ScaleLinear, scaleLinear } from 'd3-scale';
+import { line, curveLinear, Line } from 'd3-shape';
+
+import { BaseChartComponent } from './../../common/base-chart.component';
+import { ColorHelper } from '../../common/color.helper';
+
+@Component({
+  selector: 'ngx-charts-radial-gauge',
+  templateUrl: './radial-gauge.component.html',
```

Comment:
The rest of the project looks to be using inline strings via `template:`, so this looks like it would be the first usage of `templateUrl`.

I like template files broken out. That seems to go with the style guide:
https://angular.io/guide/styleguide#style-05-04

However, I think there's performance issues as SPA projects grow large:
https://medium.com/@frosty/angularjs-template-vs-templateurl-cdde055b7907

and since this is a downstream library it is avoiding the extra network requests by using inline strings intentionally.